/**:
  ros__parameters:
    use_detected_objects: False
    use_cluster: True
    use_box: True
    max_cloud_size: 55000
    downsample: False
    use_lfit: True
    use_z: True
    cluster:
      frame_id: "base_link"
      min_cluster_size: 10
      max_num_clusters: 256
      min_cluster_threshold_m: 0.5
      max_cluster_threshold_m: 1.5
      threshold_saturation_distance_m: 60.0
    hash:
      min_x: -130.0
      max_x:  130.0
      min_y: -130.0
      max_y:  130.0
      side_length: 1.0
    voxel:
      min_point:
        x: -130.0
        y: -130.0
        z: -130.0
      max_point:
        x: 130.0
        y: 130.0
        z: 130.0
      voxel_size:
        x: 0.2
        y: 0.2
        z: 0.2
